doc/uftrace-tui.md

```diff
@@ -23,10 +23,16 @@ the initial data loading.
 TUI OPTIONS
 ===========
 -f *FIELD*, \--output-fields=*FIELD*
-:   Customize field in the output.  Possible values are: total, self and addr.
-    Multiple fields can be set by using comma.  Special field of 'none' can be
-    used (solely) to hide all fields.  Default is 'total'.
-    See `uftrace-graph`(1) for an explanation of fields.
+:   Customize fields in the output. This option basically indicates graph fields.
```

Comment:
By convention in uftrace docs, we give 2 white spaces after the period.
```suggestion
:   Customize fields in the output.  This option basically indicates graph fields.
```
Please apply the same for the rest of other sentences.